pyproject.toml

```diff
@@ -21,7 +21,7 @@ line_length=120                # corresponds to -w  flag
 multi_line_output=3            # corresponds to -m  flag
 include_trailing_comma=true    # corresponds to -tc flag
 skip_glob = '^((?!py$).)*$'    # this makes sort all Python files
-known_third_party = ["arrow", "dacite", "more_itertools", "requests"]
+known_third_party = ["arrow", "dacite", "requests"]
 
 [tool.poetry.dependencies]
 python = ">=3.7,<3.11"
```

Comment:
You need to specify the version of utils which includes the uploader extractor :)

Reply:
Oh! Of course.